03-bundling/06-vite/06-typescript/vite.config.ts

```diff
@@ -1,6 +1,12 @@
 import { defineConfig } from "vite";
 import checker from "vite-plugin-checker";
+import typescript from "@rollup/plugin-typescript";
 
 export default defineConfig({
   plugins: [checker({ typescript: true })],
+  build: {
+    rollupOptions: {
+      plugins: [typescript({ noEmitOnError: false })],
+    },
+  },
 });
```

Comment:
Some of these changes are documented in README but they were not added to final files in repo. This file is a proof of it.